Sources/TuistKit/Commands/Cache/CacheWarmCommand.swift

```diff
@@ -10,6 +10,17 @@ struct CacheWarmCommand: ParsableCommand {
                              abstract: "Warms the local and remote cache.")
     }
 
+    @OptionGroup()
+    var options: CacheOptions
+
+    func run() throws {
+        try CacheWarmService().run(path: options.path, profile: options.profile, xcframeworks: options.xcframeworks)
+    }
+}
+
+// MARK: - Options
+
+struct CacheOptions: ParsableArguments {
```

Comment:
Nitpick: What about having `CacheOptions` in its own file `CacheOptions.swift`?

Reply:
The caching options has been extracted to a separate file 👍